src/github.com/getlantern/golog/golog.go

```diff
@@ -144,7 +157,15 @@ func (l *logger) print(out io.Writer, skipFrames int, severity string, arg inter
 }
 
 func (l *logger) printf(out io.Writer, skipFrames int, severity string, message string, args ...interface{}) {
-	_, err := fmt.Fprintf(out, severity+" "+l.linePrefix(skipFrames)+message+"\n", args...)
+	buf := bufferPool.Get()
```

Comment:
While I was in here, I switched to using a pooled ByteBuffer to construct our log statements, which is both more efficient than string concatenation and also integrates nicely with ctx.PrintTo.